TASK/SetDriveTaskDrivingStyle.md

```diff
@@ -8,6 +8,8 @@ ns: TASK
 void SET_DRIVE_TASK_DRIVING_STYLE(Ped ped, int drivingStyle);
 ```
 
+Sets the driving style for a ped currently performing a driving task.
+
 ```
 This native is used to set the driving style for specific ped.  
```

Comment:
You can remove the native docs here since you cover everything they said anyways

Reply:
I wanted to add an example before removing, as well keeping the link for anyone researching. After looking at the link however, it made more sense to incorporate the info into the function description.